recipes/automap/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set author = "brandtbucher" %}
+{% set name = "automap" %}
+{% set version = "0.4.8" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 51c2c7ee619714bc0fcf20d4368de25336bb63058ff8f4d8315f45cf6ea17cc8
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - automap
+
+about:
+  home: https://github.com/brandtbucher/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
+  summary: 'High-performance autoincremented integer-valued mappings.'
+  dev_url: https://github.com/{{ author }}/{{ name }}
+
+extra:
+  recipe-maintainers:
+    - {{ author }}
```

Comment:
This is going to break the bot and other parts. Please remove the author jinja2 variable